.clang-format

```diff
@@ -0,0 +1,203 @@
+# This file is generated by the following command
+# clang-format(v12.0.1, in vscode v1.56.0) -style=Google -dump-config > .clang-format
```

Comment:
I haven't used clang-format before, but based on [this answer](https://stackoverflow.com/a/46374122), shouldn't the option in line 68 of `check_tidy.py` should be `-style=file`? (Although there is no difference in this case since `.clang-format` is generated with `-style=Google`)

Other than that, LGTM!

Reply:
OMG! you are right

Reply:
I fix it and I replaced .clang-format with new one created with clang-format 11